test_suite/common/gap_utils.py

```diff
@@ -200,37 +201,60 @@ def __init__(self):
         self.size = self.dataSize + 2
 
 
-def gap_connect(central, peripheral):
+def start_advertising_of_type(peripheral, advertising_type):
+    # params
+    peripheral.advParams.setType(advertising_type)
+    peripheral.advParams.setPrimaryInterval(100, 100)
```

Comment:
Please provide other values for non connectable advertising, ST controllers have some difficulties with thos. 
I suggest `200, 400` when the advertising type is not connectable.